src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -64,69 +64,95 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         co_await winrt::resume_foreground(Dispatcher());
 
-        // "remove" all profile-related NavViewItems
-        // LOAD-BEARING: use Visibility here, instead of menuItems.Remove().
-        //               Remove() works fine on NavViewItems with an hstring tag,
-        //               but causes an out-of-bounds error with Profile tagged items.
-        //               The cause of this error is unknown.
+        // Deduce information about the currently selected item
+        std::optional<IInspectable> selectedItemTag{ std::nullopt };

```

Comment:
iinspectable is already nullable, it doesn't need to be made optional on top of that